storage/innobase/trx/trx0trx.cc

```diff
@@ -124,6 +124,7 @@ trx_init(
 	trx_t::state here to NOT_STARTED. The FORCED_ROLLBACK
 	status is required for asynchronous handling. */
 
+	trx->id_saved = trx->id;
```

Comment:
I am not sure if that's the right place to save the ID - could this result in old committed RW trx IDs reused by new transactions in the slow log?